reader.lua

```diff
@@ -277,7 +277,7 @@ if ARGV[argidx] and ARGV[argidx] ~= "" then
         local home_dir =
             G_reader_settings:readSetting("home_dir") or ARGV[argidx]
         UIManager:nextTick(function()
-            FileManager:showFiles(home_dir)
+            FileManager:showFiles(home_dir, "", true)
```

Comment:
`nil` instead of `""`, or does something wonky happen?

Reply:
> `nil` instead of `""`, or does something wonky happen?

Didn't check but assumed so.
Not relevant anyways now